### ISSUE TITLE
Assets not working correctly

### ISSUE THREAD

**Issue by SladkiyTort** [ruffle-rs/ruffle-android#151]
Some games has assets in their files. When I start swf it works fine, but it is separated for a several other swf, and folder. (Folder path example below
- data
- index.html
- - 01.html
- - 01.swf
- - 02.html
- - 02.swf
(And etc))
So the thing is, when I start "01.swf" the button that answers for changing game to "02.swf" do not works, somehow the "Flash Game Player Classic" manages to change swf's (but its lags very hard).
And also I want to say that ruffle is great (as always), and every new update improves every aspects of it, well done.

**Comment by torokati44** [comment]
Yes, we are aware, sorry. Probably "Ruffle Bundles" will be the way to solve this.
